src/linux/elf.rs

```diff
@@ -443,8 +446,13 @@ impl Mergeable for ElfInfo {
 
                 match left.symbols.entry(*addr) {
                     btree_map::Entry::Occupied(mut e) => {
-                        // we already have one so just discard this one
-                        e.get_mut().is_multiple = true;
+                        if e.get().is_synthetic && !sym.is_synthetic {
```

Comment:
nit: I think having a `if sym.is_synthetic  { continue }` before would help to have something more clear.

Reply:
Good point, will do!